src/app/user/controller.js

```diff
@@ -16,12 +16,10 @@ const createTokenPayload = user => {
 };
 
 const createUser = async (req, res) => {
-  const { email, username, password, firstName, lastName, role } = req.body;
+  const { email, username, password } = req.body;
```

Comment:
Please add role back here @Chiagozielam

Reply:
okay, I'm just trying to figure out how to add this without pushing my other staged changes

Reply:
Is this not in a different branch?